### ISSUE TITLE
8416 - 508 Audit - Defect 1 - Critical: Privacy Act headings not read by screen reader

### ISSUE THREAD

**Issue by meaghan-sullivan** [department-of-veterans-affairs/va.gov-team#130357]
<html>
<body>

Defect 1 \| Critical \| Created:WCAG 1.3.1 Information and Relationships \| Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.On the Privacy Act Statement screen (Privacy Act Statement), "Respondent Burden:" and “Privacy Act Notice:” are not read as headings by assistive technology. (New: 01/16/2026)On the Submit Medical Expenses To Support A Pension Or DIC Claim \| Veterans Affairs screen, after activating Feedback, the structure of information cannot be understood by JAWS. There are two main regions and it is possible to read outside the modal with the arrow keys. (New: 01/16/2026)
--

</body>
</html>

**Comment by timvang-hrl** [comment]
On the JAWS virtual-cursor issue, the modal is rendered by the shared `va-omb-info` design-system component, so the aria/focus behavior lives there. Fixing it needs a design-system update to add proper aria-modal/background hiding and focus trapping to va-omb-info. If we stick with that component, the change has to come from the DS team.
